tests/functional/monitoring/prometheus/metrics/test_monitoring_defaults.py

```diff
@@ -282,3 +283,43 @@ def test_provider_metrics_available(threading_lock):
         "so that the list of metrics without results is empty."
     )
     assert list_of_metrics_without_results == [], msg
+
+
+@blue_squad
+@tier1
+@bugzilla("2297285")
+@pytest.mark.polarion_id("OCS-XXX")
+def test_monitoring_ipv6(threading_lock):
```

Comment:
no `rook-ceph-exporter` on External mode clusters because rook does not manage ceph in this case. 
I think this test is irrelevant to External mode and we may add "skip_if" decorator

Reply:
done